contracts/v2/paranets/ParanetNeuroIncentivesPool.sol

```diff
@@ -7,9 +7,15 @@ import {ParanetsRegistry} from "../storage/paranets/ParanetsRegistry.sol";
 import {HubV2} from "../Hub.sol";
 import {ParanetErrors} from "../errors/paranets/ParanetErrors.sol";
 import {ParanetStructs} from "../structs/paranets/ParanetStructs.sol";
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {
+    EMISSION_MULTIPLIER_SCALING_FACTOR,
+    PERCENTAGE_SCALING_FACTOR,
+    VOTERS_WEIGHTS_SCALING_FACTOR
```

Comment:
Hmmm. Yeah, this might be the case. Then I would just check if total weight doesn't exceed 100% and rest would be registrar's responsibility to assign correct weights